plugins/modules/group.py

```diff
@@ -40,38 +40,39 @@
     - Specifies the action to be performed.
     - C(present) creates a new group. When the group already exists, use I(sate=modify) to change
       its attributes.
-    - C(absent) deletes an existing group. Users who are group members are not removed.
-    - C(modify) changes specified value of attributes of an existing group. When the group does not
+      Changes specified value of attributes of an existing group. When the group does not
```

Comment:
This line should go with C(present) only, in a better manner. Right now, It can create a little confusion.